assets/css/dashboard/new-pa-message-page/associate-alert-page.scss

```diff
@@ -0,0 +1,220 @@
+.sidebar-container:has(
+    + .page-content > .new-pa-message > .associate-alert-page
+  ) {
+  display: none !important;
+}
+
+.associate-alert-page {
+  padding: 24px 48px;
+}
+
+.associate-alert-page-header {
+  padding-right: 48px;
+  max-width: 1380px;
+
+  &__cancel {
+    flex: none;
+    width: 100px;
+
+    button {
+      margin: 8px 16px 8px 16px;
+      height: 48px;
+      color: $text-link-primary;
+      text-decoration: none;
+
+      &:hover {
+        color: $text-link-primary;
+        background-color: $cool-gray-15;
+      }
+    }
+  }
+}
+
+.associate-alert-page-body {
+  flex-wrap: nowrap;
+  padding-right: 48px;
+}
+
+.associate-alert-filter-selection {
+  flex: none;
+  font-size: 20px;
+  width: 200px;
+  padding: 0px 12px;
+  margin-top: 40px;
+
+  &__label {
+    margin-bottom: 8px;
+  }
+
+  &__button-group {
+    margin-bottom: 48px;
+  }
+
+  &__selected {
+    color: #ffffff !important;
+    background: #d9d9d93d !important;
+  }
+
+  button {
+    text-align: left;
+    height: 38px;
+    min-width: 200px;
+    color: $text-secondary;
+    background-color: $cool-gray-20;
+    border: none;
+  }
+}
+
+.associate-alert-table {
+  margin: 40px;
+  max-width: 1096px;
+
+  th,
+  td {
+    padding: 8px 16px;
+    border-bottom: 1px solid $cool-gray-40;
+    vertical-align: top;
+  }
+
+  th {
+    padding-top: 24px;
+    height: 56px;
+  }
+
+  &__row {
+    background-color: $cool-gray-15;
+
+    &:hover {
+      background-color: $cool-gray-20;
+    }
+  }
+
+  &__message {
+    min-width: 300px;
+  }
+
+  &__id {
+    width: 90px;
+  }
+
+  &__start-end {
+    min-width: 200px;
+    padding-right: 0px;
+  }
+
+  &__last-modified {
+    min-width: 220px;
+  }
+
+  &__select {
+    min-width: 102px;
+
+    button {
+      color: $text-secondary;
+      height: 24px;
+
+      &:hover {
+        color: $text-secondary;
+        opacity: 84%;
+      }
+    }
+  }
+
+  &__empty {
+    font-weight: bold;
+    color: white;
+    padding-top: 16px;
+    padding-left: 16px;
+  }
+}
+
+.alert-description {
+  padding: 16px;
+
+  &__header {
+    margin-bottom: 8px;
+    justify-content: center;
+  }
+
+  &__header-id-label {
+    display: inline;
+    font-size: 20px;
+    font-weight: 500;
+  }
+
+  &__header-id {
+    display: inline;
+    font-size: 16px;
+    font-weight: 400;
+  }
+}
+
+.alert-items-modal {
+  color: white;
+
+  .modal-content {
+    background-color: $cool-gray-20;
+    min-width: 800px;
```

Comment:
I noticed that the modal looks a bit off centered horizontally and I think it's because the parent container is not changing its width based on the children. It looks like React Bootstrap adds a `.modal-dialog` element between `<Modal>` and its first child by default. If you move the `min-width` in `.modal-content` to `.modal-dialog` it should handle centering a bit better.

Reply:
ah, nice thank you!